component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -88,6 +88,14 @@ public static String decrypt(String cipherText) throws CryptoException {
 		return new String(CryptoUtil.getDefaultCryptoUtil().base64DecodeAndDecrypt(cipherText), Charsets.UTF_8);
 	}
 
+	public static String getTOTPIssuerDisplayName(String tenantDomain) {
+		String issuer = getTOTPParameters().get(TOTPAuthenticatorConstants.TOTP_ISSUER);
```

Comment:
The "issuer" parameter will get from local file (application-authentication.xml). But for tenant there are two ways to get these properties. 
- we can get these properties from local file (application-authentication.xml).
- we can load this file in the registry and get the properties from authentication context.
 Hence, don't we need to check the case if we load this xml file in the registry. Is it handle in this PR or anywhere else?

Reply:
I need to configure a common Issuer. BTW, extended to supporting tenant wise.